Camera.cpp

```diff
@@ -11,5 +11,17 @@ Camera::Camera(std::string subtype)
     : Object(ospNewCamera(subtype.c_str()), subtype)
 {}
 
+void Camera::setParam(const char *_id, ANARIDataType type, const void *mem)
+{
+  std::string id(_id);
+
+  if (id == "imageRegion") {
+    Object::setParam("imageStart", ANARI_FLOAT32_VEC2, (const float *)mem);
+    Object::setParam("imageEnd", ANARI_FLOAT32_VEC2, ((const float *)mem)+2);
+  }
```

Comment:
With an `else` we avoid the "unknown param 'imageRegion'" error from OSPRay.

Reply:
Yes indeed. I realized that, too, but forgot to add to the patch. Could you just add the `else` there or should I?

Reply:
I'll do it. In general we will merge to the internal repo first and then push back to GitHub.